op-node/p2p/gossip_test.go

```diff
@@ -153,9 +153,11 @@ func TestBlockValidator(t *testing.T) {
 	// Params Set 2: Call the validation function
 	peerID := peer.ID("foo")
 
-	v2Validator := BuildBlocksValidator(testlog.Logger(t, log.LevelCrit), cfg, runCfg, eth.BlockV2)
-	v3Validator := BuildBlocksValidator(testlog.Logger(t, log.LevelCrit), cfg, runCfg, eth.BlockV3)
-	v4Validator := BuildBlocksValidator(testlog.Logger(t, log.LevelCrit), cfg, runCfg, eth.BlockV4)
+	// Create a mock gossip configuration for testing
+	mockGossipConf := &mockGossipSetupConfigurables{}
```

Comment:
You could deduplicate this a fair bit by just using `&mockGossipSetupConfigurablesWithThreshold{threshold: 60 * time.Second}` and removing the `mockGossipSetupConfigurables` type?